### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@rgl FYI

File: README.md [hunk 1]
```diff
@@ -93,6 +93,19 @@ For projects hosted by [GitLab](https://gitlab.com) reference [Microsoft.SourceL
...
 
+If your project is hosted by GitLab older than version 12.0 you must specify `SourceLinkGitLabHost` item group in addition to the package reference:
+
+```xml
+<ItemGroup>
+  <SourceLinkGitLabHost Include="gitlab.yourdomain.com" Version="11.0"/>
+</ItemGroup>
+```
+
+The item group `SourceLinkGitLabHost` specifies the domain of the GitLab host and the version of Bitbucket.
```
Comment (rgl) [inline README.md:104]: Can you please rename to GitLab?

**Review by rgl — COMMENTED** [review]
Nice! Thank you for adding this!

File: README.md [hunk 2]
```diff
@@ -93,6 +93,19 @@ For projects hosted by [GitLab](https://gitlab.com) reference [Microsoft.SourceL
 </ItemGroup>
 ```
 
+If your project is hosted by GitLab older than version 12.0 you must specify `SourceLinkGitLabHost` item group in addition to the package reference:
+
+```xml
+<ItemGroup>
+  <SourceLinkGitLabHost Include="gitlab.yourdomain.com" Version="11.0"/>
```
Comment (rgl) [inline README.md:100]: Is the property used to set the hostname really called Include?

Comment (tmat) [reply README.md:100]: Yes. `Include` adds item to `ItemGroup`.

File: README.md (hunk 1 above)
Comment (tmat) [reply README.md:104]: Good catch

**Pull request merged**